src/components/PhotoDetails.vue

```diff
@@ -147,10 +147,12 @@ export default {
         if (image.creator) {
           if (image.creator_url) {
             byCreator = `by <a href="${image.creator_url}">${image.creator}</a>`;
-          } else {
+          } 
```

Comment:
These changes are not required and can potentially introduce merge conflicts in other pending PRs that change this file.

Reply:
I did this to supress he eslint warnings. I will revert this back.